scripts/release.sh

```diff
@@ -61,7 +61,7 @@ if ! release=$(gh api \
   echo "Failed to create release: $release"
 fi
 
-releaseId=$(jq .id <<< "$release")
```

Comment:
I recently learned that `  -sr, --space-redirects   redirect operators will be followed by a space` can retain the original style (which I like a little more)

Reply:
Sounds good, done!